peribolos/knative-sandbox.yaml

```diff
@@ -655,9 +655,8 @@ orgs:
           Knative Release Leads:
             description: Active member of the Knative Release Leads rotation.
             maintainers:
-            - tcnghia
-            - vaikas
-            - fallback-notification-blocker
```

Comment:
I think this should stay in until we remove all CODEOWNERS things.

Reply:
lol, I just wanted to make it consistent :) I guess I went the wrong way :) 
https://github.com/knative/community/blob/main/peribolos/knative.yaml#L873